styles/card.css

```diff
@@ -0,0 +1,155 @@
+.card {
+    perspective: 150rem;
+    height: 50rem;
+    max-width: 30rem;
+    position: relative;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+
+    @media screen and (min-width:800){
+    
+    }
+
+}
+
+.card__side {
+    border-radius: .3rem;
+    box-shadow: 0 1.5rem 4rem #00000033;
+    height: 50rem;
+    transition: all .2s ease;
+    min-width: 30rem;
+    backface-visibility: hidden;
+    overflow: hidden;
+    position: absolute;
+}
+
+.card__side--front {
+    background-color: #fff;
+}
+
+.card__side--back {
+    transform: rotateY(180deg);
+}
+
+.card:hover .card__side--front {
+    transform: rotateY(-180deg);
+}
+
+.card:hover .card__side--back {
+    transform: rotateY(0);
+}
+
+.card--india {
+    background-image: linear-gradient(to bottom, #ffb900, #fff, green);
+}
+
+.card--uganda {
```

Comment:
should  we be  using variable here?

Reply:
since i use this once, we don't have to.